fcrepo-clients/src/main/java/edu/unc/lib/dl/fcrepo4/BinaryObject.java

```diff
@@ -47,7 +47,7 @@ protected BinaryObject(PID pid, Repository repository, RepositoryObjectDataLoade
 		super(pid, repository, dataLoader);
 		
 		metadataUri = URI.create(pid.getRepositoryUri().toString()
-				+ "/" + RepositoryPathConstants.FCR_METADATA);
+				+ RepositoryPathConstants.FCR_METADATA);
```

Comment:
This was just added in the last cleanup commit, I'm a little worried about the impact of it.  Could you change this using the URIUtil.join() method to put the pieces together to ensure that there is always a slash there?